tests/modules/groups/test_groups.py

```diff
@@ -159,6 +159,34 @@ def test_create_pos_holders(client):
     }]
 
 
+def test_create_group(client):
+    group_id = helpers.create_group("Page House", "May the work be light",
+                                    "House", False, False, True, True, True)
+    assert helpers.get_group_data(group_id, [
+        "group_id", "group_name", "group_desc", "type", "anyone_can_send",
+        "members_can_send", "newsgroups", "visible", "admin_control_members"
+    ]) == {
+        "group_name": "Page House",
+        "group_desc": "May the work be light",
+        "type": "House",
+        "newsgroups": 0,
+        "anyone_can_send": 0,
+        "members_can_send": 1,
+        "visible": 1,
+        "admin_control_members": 1,
```

Comment:
Seems a little counter-intuitive that these values are booleans when passed to `create_group()` but integers in the result of `get_group_data()`. Would be nice to convert them back to booleans.